token/cli/src/main.rs

```diff
@@ -305,6 +305,16 @@ pub fn signers_of(
     }
 }
 
+fn check_decimals(  
+    decimals: u8,
+) -> bool {
+    let d = decimals;
+    if d> 20 {
+    return true;
+    };
+    return false;
+}
```

Comment:
Sorry, I'm not sure that we need this check in the end -- can we remove it and consider adding it in a new PR with a flag to toggle the check?